drud/ddev
# Update mkcert for catalina compatibility

## The Problem/Issue/Bug:

mkcert v1.3.0 is incompatible with up-and-coming macOS Catalina

This is probably blocked by https://github.com/FiloSottile/mkcert/issues/192 - linuxbrew mkcert doesn't work right?

## How this PR Solves The Problem:

Upgrade to mkcert v1.4.0

## Manual Testing Instructions:

On Catalina, 
* `ddev poweroff`
* `ddev start`
* Make sure the https link created is valid in Chrome

## Automated Testing Overview:


## Related Issue Link(s):

## Release/Deployment notes:
